declarations/TikTok.json

```
@@ -3,10 +3,11 @@
   "documents": {
     "Terms of Service": {
       "fetch": "https://www.tiktok.com/legal/terms-of-use?lang=en",
-      "select": {
-        "startBefore": "#terms-eea",
-        "endBefore": "#terms-in"
-      }
+      "select": ".main-body",
```

Version should be cleanest with the following suggested selector:
```suggestion
      "select": "article",
```